lib/tasks/remove-files.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// Creates a directory with the name directoryName in cwd and then sets cwd to
+// this directory.
+
+var Promise = require('../ext/promise');
+var Task    = require('../models/task');
+var _       = require('lodash');
+var chalk   = require('chalk');
+
+module.exports = Task.extend({
+
+  init: function() {
+    this.rimrafSync  = this.rimrafSync || require('rimraf').sync;
+  },
+
+  // Options: String directoryName, Boolean: dryRun
+
+  run: function(options) {
+    var ui = this.ui;
+    var rimrafSync = this.rimrafSync;
+
+    var paths = this.paths = options.paths;
+    if (!_.isArray(paths)) {
+      paths = [paths];
+    }
+
+    paths = _.compact(paths);
+    if (!paths.length) {
+      throw new Error('No file paths specified to remove');
+    }
+
+    return Promise.resolve().then(function() {
+        paths.forEach(function(dirToRemove) {
+          if (!options.dryRun) {
+            rimrafSync(dirToRemove);
+          }
+
+          var withTrailingSlash = ensureWithTrailingSlash(dirToRemove);
```

Comment:
Currently `remove-files` is not the best name for this task. 
It assumes that all the files are directories and adds trailing slash for each file name in output.